aikau/src/main/resources/alfresco/renderers/InlineEditProperty.js

```diff
@@ -317,6 +318,10 @@ define(["dojo/_base/declare",
                   widgets: [primaryFormWidget].concat(autoSetFields)
                }
             }, this.formWidgetNode);
+            // NOTE: This line is specifically required to support Firefox, for some reason the standard
+            //       key handling is being suppressed, this was uncovered on the move from Dojo 1.9.0 to
+            //       both 1.9.6 and then 1.10.4
+            query(".alfresco-forms-controls-BaseFormControl .control input").on("keypress", lang.hitch(this, this.onValueEntryKeyPress));
          }
```

Comment:
This line can potentially target controls other than just the InlineEditProperty, given the query. Is this intended, and if so, is this the best place for the code?